pubsub/responses.go

```diff
@@ -28,6 +28,9 @@ const (
 // AppResponse is the object describing the response from user code after a pubsub event.
 type AppResponse struct {
 	Status AppResponseStatus `json:"status"`
+
+	// When Retry/Drop, this field should be populated with the error message to be published to DLQ if configured.
+	Error error `json:"error"`
```

Comment:
Was this intentionally added? I don't quite see where this is used- at least for this PR.

Reply:
Oops that bled from a different PR I'm working on. Good catch!